data/json/vehicle_parts.json

```diff
@@ -37,6 +37,29 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "reclining_seat",
+        "name" : "reclining seat",
+        "symbol" : "#",
+        "color" : "red",
+        "broken_symbol" : "*",
+        "broken_color" : "red",
+        "damage_modifier" : 60,
+        "durability" : 100,
+        "item" : "seat",
+        "difficulty" : 3,
+        "size" : 25,
+        "location" : "center",
+        "flags" : [
+            "BED", "SEAT", "BOARDABLE", "BELTABLE", "CARGO"
```

Comment:
How about I split the warmth bonus in half and keep the sleep bonus? Sleepy bonus is integer 3 vs 4, can't do 3.5 :(